src/interfaces/keyIterator.ts

```diff
@@ -25,6 +25,8 @@ export interface KeyIterator {
     label: string
     privateKey: string
     dedicatedToOneSA: boolean
-    meta: null
+    meta: {
+      timestamp: number
+    }
```

Comment:
To be consistent with the old data, that would not have a timestamp, the type should be `number` or `null` maybe?